web/src/app/services/classification/classification.service.ts

```diff
@@ -0,0 +1,63 @@
+import {Injectable} from '@angular/core';
+import {HttpClient, HttpHeaders} from '@angular/common/http';
+import {Observable} from 'rxjs/Observable';
+import {environment} from '../../../environments/environment';
+import {DatePipe} from '@angular/common';
+import {AlertService} from '../alert/alert.service';
+import {Classification} from '../../models/classification';
+import {AlertModel, AlertType} from '../../models/alert';
+import {saveAs} from 'file-saver/FileSaver';
+
+@Injectable()
+export class ClassificationService {
+
+  httpOptions = {
+    headers: new HttpHeaders({
+      'Content-Type': 'application/json',
+      'Authorization': 'Basic VEVBTUxFQURfMTpURUFNTEVBRF8x'
+    })
+  };
+
+  constructor(private httpClient: HttpClient, private alertService: AlertService) {
+  }
+
+  // GET
+  getClassificationDomains(): Observable<string[]> {
+    return this.httpClient.get<string[]>(environment.taskanaRestUrl + '/v1/classifications/domains', this.httpOptions);
+  }
+
+  // GET
+  exportAllClassifications() {
+    this.httpClient.get<Classification[]>(environment.taskanaRestUrl + '/v1/classificationdefinitions', this.httpOptions)
```

Comment:
I would use only 1 method exportClassifications, if classification is defined, then add clasification to the url otherwise export all